metal/resource_metal_device_test.go

```diff
@@ -117,13 +117,14 @@ func TestAccMetalDevice_Basic(t *testing.T) {
 		CheckDestroy: testAccCheckMetalDeviceDestroy,
 		Steps: []resource.TestStep{
 			{
-				Config: testAccCheckMetalDeviceConfig_basic(rs),
+				Config: testAccCheckMetalDeviceConfig_minimal(rs),
 				Check: resource.ComposeTestCheckFunc(
 					testAccCheckMetalDeviceExists(r, &device),
 					testAccCheckMetalDeviceNetwork(r),
-					testAccCheckMetalDeviceAttributes(&device),
```

Comment:
testAccCheckMetalDeviceAttributes expects the `tfacc-test-device` hostname, so we push this check to the second step.